ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Exif.java

```diff
@@ -36,12 +42,18 @@ private Exif(@NonNull TiffOutputSet tiffOutputSet) {
     }
 
     @NonNull
-    public static Builder builder() throws ImageWriteException {
-        return new Builder();
+    public static Builder builder(@NonNull final byte[] jpeg)
```

Comment:
Requiring the jpeg byte array beforehand now, because we need the byte order for the new EXIF Directory we create to override the existing one. 

In the new `PhotoTest#should_supportParceling()` test I detected that after unparceling the Photo the required EXIF Tags are not read. After fixing that I saw that the byte order was wrong when reading the updated Tags. 

These bugs went unnoticed because Photo parceling is not used currently. I considered removing parcelability, but we might need it for persisting fragment state in the future (regarding app restarts for example).